tests/lib/rules-preprocessor/gjs-gts-parser-test.js

```diff
@@ -91,6 +91,10 @@ const valid = [
       <template>
         <div {{on 'click' noop}} />
       </template>
+
+      <template>
+        <div {{on 'click' noop}} />
```

Comment:
@bmish this is the bad code -- can't have two default templates

Reply:
@bmish it would have failed to parse it without this change

Reply:
Can you provide some context about the actual fix too? What was wrong before and how does your change fix it?

Reply:
Before it would have converted the template parts to template string literals.
```js
`
Tpl1
`
`
Tpl2
`
```
But that makes it  one tagged template literal... Instead of 2 separates.
Now i make them tagged already.

```js
""
`
Tpl1
`
""
`
Tpl2
`
```
I use string as tag name, as otherwise eslint would complain about missing references to identifiers